src/s2a_moms_tent.tw2

```diff
@@ -112,12 +112,12 @@ Mom loves to keep notes. She writes down all kinds of things — keeps track of
 
 ::Mom's Tent - notepad 2
 It was a to-do list. No real surprises there:
-* REMEMBER: working extra shifts for Jolene next Sat. & Sun.
+[(live: 1s)[(if: time > 2s)[$dissolve[* REMEMBER: working extra shifts for Jolene next Sat. & Sun.
```

Comment:
This `live:` fixes HTML bullets displaying immediately.

Reply:
The bullets don't display immediately, but they do still pop in before the text that's after the first set of bullets and after the final bullets. I think you'll have to wrap each part of the passage in live: macros to avoid the bullets coming in weird, kind of like I set up the rap sections.

Reply:
Weird, I had this working on my end. 1s.

Reply:
Yeah, this is working fine in Firefox. Let me try Chrome.

Reply:
Ugh, wow, yeah, this is broken in Chrome. Ugh.

Reply:
lmfao the game doesn't even open in Safari.

Reply:
Reopened https://github.com/whymog/efjm/issues/15